lib/daru/category.rb

```diff
@@ -64,6 +62,17 @@ def name= new_name
       self
     end
 
+    def plotting_library= lib
+      case lib
+      when :gruff
+        @plotting_library = lib
+        extend Daru::Plotting_Gruff::Category if Daru.has_gruff?
```

Comment:
Naming it to `Daru::Plotting::Category::Nyaplot` is leading to `NameError`s. 
```ruby
      NameError:
        uninitialized constant Daru::Plotting::Category::Nyaplot::Plot

```
That's why I have renamed it `Daru::Plotting::Category::NyaplotLibrary` and same for Gruff.